shell/platform/android/test/io/flutter/embedding/engine/renderer/FlutterRendererTest.java

```diff
@@ -533,15 +533,15 @@ public void ImageReaderSurfaceProducerSkipsFramesWhenResizeInflight() {
     // Resize.
     texture.setSize(4, 4);
 
-    // Let callbacks run.
+    // Let callbacks run. The rendered frame will manifest here.
     shadowOf(Looper.getMainLooper()).idle();
 
     // We should not get a new frame because the produced frame was for the previous size.
```

Comment:
Stale comment?

Reply:
This comment is correct the comment below is wrong and I've fixed it.